### DISCUSSION TITLE
useStore hook , React

### DISCUSSION THREAD

**Discussion by sudokud** [TanStack/form#1056] [Q&A] [answered]
hello,
i'm trying to import the useStore hook i got this error Module '"@tanstack/react-form"' has no exported member 'useStore'
anyone have an ides on how to use this hook ?

![Screenshot 2024-12-05 at 7 25 10 pm](https://github.com/user-attachments/assets/a8dbf698-0448-4f64-882e-eaf014f52fb4)

thanks

**Comment by crutchcorn** [comment, marked as answer]
Oh that's gotta be a bug on our end.

Try `npm i`-ing `@tanstack/react-store` and importing from there in the short-term

**Reply by sudokud** [reply]
thank for your quick feedback, i'll try that

**Reply by sudokud** [reply]
when import the useStore from @tanstack/react-store its create the following error 
"Import declaration conflicts with local declaration of 'useStore'."
it works fine when i import it with as and change the name 
useStore from @tanstack/react-form is available somehow in the namespace but cannot be used or i dont know how to use it

thanks @crutchcorn
